its/src/test/java/com/sonar/it/csharp/RazorClassLibProjectTest.java

```diff
@@ -44,12 +43,11 @@ public class RazorClassLibProjectTest {
   private static final String S6798_FOLDER = "RazorClassLib:S6798";
   private static final String S6800_FOLDER = "RazorClassLib:S6800";
   private static final String S6802_FOLDER = "RazorClassLib:S6802";
-  private static final String SONAR_RULE_S6802 = "csharpsquid:S6802";
-  private static final String S6802_COMPONENT_RAZOR_FILE = "RazorClassLib:S6802/S6802.razor";
-  private static final String S6802_COMPONENT_CS_FILE = "RazorClassLib:S6802/S6802.cs";
 
   @BeforeAll
   public static void beforeAll() throws Exception {
+    ORCHESTRATOR.getServer().provisionProject(PROJECT, PROJECT);
+    ORCHESTRATOR.getServer().associateProjectToQualityProfile(PROJECT, "cs", "blazor_rules");
```

Comment:
Can we add a comment explaining why we are doing that (i.e. to activate a rule which would be otherwise deactivated, since it's not in SonarWay)?